### PR TITLE
Portfoliobranch2

### PR DESCRIPTION
**Pull request by EmekaEmmanuel**
Portfolio Setup is a simple portfolio website with these sections contained in this project;
- Toolbar
- Headline section (right after the header).
- Works
- About myself
The layout was done using css flexbox and css grid, button interactions implemented.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+.works_section {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 10px;
+  width: 90%;
+  margin: auto;
+  padding: 20px 0 30px 0;
+}
+
```
Comment (BregornOriginal) [inline index.css:94]: - [ ] Your projects look great on 375px, but when the width is bigger the responsive until desktop version 768 px it's lost. To fix this we can improve your article class given a `display: flex;
 flex-direction: column;
 align-items: center;`

In this way, you will have your card's article centered in the middle whatever the width of the view is.

***This is how your cards look until with a wide 375px or more***

image


***Here it's your card with the property is that I mention at the beginning*** 💯💯💯💯 

image

File: index.css [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+.about_self_section {
+  width: 90%;
+  margin: auto;
+  padding: 40px 0 40px 0;
+}
+
+.font7 {
+  width: 295px;
+  color: #000;
+  font-size: 40px;
```
Comment (BregornOriginal) [inline index.css:258]: - [ ] This font it's too big for a mobile version and doesn't follow the FIGMA design, please put a smaller size to this section, maybe around 20px would be good. Also, you can check the exact font size on the FIGMA template in the column description on the right.

image

File: index.css [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+
+.article1 button a:hover,
+.article2 button a:hover,
+.article3 button a:hover,
+.article4 button a:hover,
+.about_self_section button a:hover {
+  color: #fff;
+}
+
+.about_self_section {
```
Comment (BregornOriginal) [inline index.css:249]: - [ ] Good work on this section. Kindly align your ul social icons at the bottom. In this way will look better and follow the same design as FIGMA. 🥇 

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @EmekaEmmanuel  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✅ Linters are passing
✅ Good pull request description
✅ README looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

### - [OPTIONAL] ***Kindly on your next project use commits in the way that you create a button and you do a commit, you create a header and do a commit. In this way, you will be able to find a bug in your application and it will be more organized to whatever person want to take a look at it. Also, your commit history would be more descriptive and simple to read.***

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @EmekaEmmanuel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good PR Description 👍 


## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**